### DISCUSSION TITLE
Version 1.2.1 does not work for me

### DISCUSSION THREAD

**Discussion by Aluerie** [SpotCompiled/SpotilifeC#4] [General]
idk if you really strive for making universally working Spotify IPA but the newly released IPA does not work for me.

There is still "Premium" tab which is not there in working julioverne releases (such as famous 8.6.94 IPA). And for example it does not let me skip as I wish.
![Untitled](https://user-images.githubusercontent.com/33165440/209120089-6418df8f-ad4b-44dd-8837-3a9b8510971a.png)

Not sure what to share about my system. It's iOS 15.7.2 on iPhone 7. I sideloaded it with Sideloadly. If I can/need provide more - please, tell me what and how (if it is some advanced stuff)

Well, if it does not work for me - it does not mean it does not work for everybody, so I hope other people have more luck.

**Comment by yodaluca23** [comment]
Did you go to the Spotilife settings and click save changes?

**Reply by Aluerie** [reply]
Yes, I clicked around with 
* Apply Changes (Close Spotify)
* Clear Cache
* Enable-Disable Spotilife
* Reinstalling after logging out 
* Closing the app right after "Thanks Julio" screen (as it suggests itself) 

and no luck
